sources/packages/backend/libs/utilities/src/sqlLoader.ts

```diff
@@ -1,7 +1,7 @@
 import * as fs from "fs";
 import * as path from "path";
 
-let sqlDirPath = "src-sql";
+let sqlDirPath = "../../libs/src-sql";
 if (process.env.ENVIRONMENT === "test") {
   sqlDirPath = "../../src-sql";
 }
```

Comment:
Not related to PR. What are we trying to do from line 5-7 @andrewsignori-aot ?

Reply:
I believe.. in test the jest dir changes